tests/test_cli.py

```diff
@@ -12,7 +12,8 @@
 trtype = "TCP"
 gateway_name = socket.gethostname()
 addr = "127.0.0.1"
-listener_list = [["-g", gateway_name, "-a", addr, "-s", "5001"], ["-s", "5002"]]
+listener_list = [["-g", gateway_name, "-a", addr, "-s", "5001"], \
+                 ["-g", gateway_name, "-a", addr, "-s", "5002"]]
```

Comment:
The point of having this list because we wanted to test two different supported cases: one when all params are specified, and another when only a port is specified. Now, when we don't support the second case, I don't think much sense in keeping this list, as you tetisting the same twice. I would  just remove ` @pytest.mark.parametrize("listener", listener_list)` in `test_create_listener` and `test_delete_listener`, so we have just one test.